python_transport/wirepas_gateway/utils/log_tools.py

```diff
@@ -16,7 +16,7 @@
 def setup_log(
     module,
     level="debug",
-    log_format="%(asctime)s | [%(levelname)s] %(name)s: %(message)s",
+    log_format="%(asctime)s | [%(levelname)s] %(name)s.%(funcName)s@%(filename)s:%(lineno)d: %(message)s",
```

Comment:
Log prefix will be quite long. I would remove funcName at least as filename/lineno already gives it.
But I'm not convince about filename. If name is well chosen, it should be quite straight forward to find the message in the code. No?

Reply:
I think the function name might be more useful as people will end up CTRL+F on all source. 

For that reason I agree that we do not need to keep the filename.

Reply:
I removed the function name but I think the filename and number is at least needed in case the function name is reused in multiple places.